mmv1/third_party/terraform/services/bigquery/iam_bigquery_member_dataset.go

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Can we add a comment somewhere indicating that this is mostly copied from the generated version of the code? And then another comment later that generally marks where the handwritten custom code is?